### ISSUE TITLE
Error when resolving anchors. No callback received.

### ISSUE THREAD

**Issue by vishal219** [google-ar/arcore-ios-sdk#100]
We are having an AR application which saves the cloud anchor ids in the backend and then retrieves it & tries to resolve the anchor.

 private func resolvedAnchors() {
        guard let deviceLastLocation = deviceLastLocation else { return }
        
        self.mementosVM.getAllMementos().forEach {[weak self] memento in
            guard let self = self else { return }
            
            if let anchorId:String = memento.anchor_id {
                printObj("AnchorId: "+anchorId)
                
                self.googleSession?.resolveCloudAnchor(anchorId: anchorId)
            }
        }
    }
    

and I don't get any failure callbacks in didFailToResolve but sometimes when it is able to fetch the anchor I do receive didResolve callback.

Used to work fine like a month ago. Suddenly the resolving is taking too long & fails to resolve multiple calls.

**Comment by 15kingben** [comment]
Hi Vishal, can you provide a minimum reproducible sample project? Also, do you see the same behavior when using the asynchronous APIs e.g. https://developers.google.com/ar/reference/ios/category/GARSession(CloudAnchors)#resolvecloudanchorwithidentifier:completionhandler:error:

**Comment by vishal219** [comment]
[Sample.zip](https://github.com/google-ar/arcore-ios-sdk/files/15060703/Sample.zip)
@15kingben 
 Here is the sample app. You will need to add your google-info.plist and api key.  I have used local storage (UserDefaults for anchor Ids) in this.

**Comment by 15kingben** [comment]
Is this issue related to https://github.com/google-ar/arcore-unity-extensions/issues/56, cloud anchors not hosting when TTL = 365?

**Comment by vishal219** [comment]
No. It's not. It is regarding slow resolving and sometimes not resolving in case of multiple anchor resolving request simultaneously.

**Comment by vishal219** [comment]
Also receiving the following state for anchors: `GARCloudAnchorStateErrorInternal` in the callback
